### PR TITLE
Update pets-training to v1alpha2

### PR DESCRIPTION
**Pull request by royxue**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/183)

### CODE REVIEW

**Comment by royxue** [comment]
@ldcastell Plz check this PR, which is with new apiVersion: v1alpha2, and be able to work.

File: object_detection/jobs/pets-training.yaml [hunk 1]
```diff
@@ -1,77 +1,61 @@
...
+      replicas: 1
+      restartPolicy: OnFailure
+      template:
+        spec:
+          containers:
+          - name: tensorflow
+            image: lcastell/pets_object_detection
+            resources:
+              limits:
+                nvidia.com/gpu: 1
```
Comment (ldcastell) [inline object_detection/jobs/pets-training.yaml]: Question, if your cluster does not have GPU, will this make the job fail?

Comment (ldcastell) [reply object_detection/jobs/pets-training.yaml]: I agree with @jlewi I have started to look into parametrizing the manifests into a ksonnet app, and address this issue too.
I will see if I can push something by next weekend

Comment (royxue) [reply object_detection/jobs/pets-training.yaml]: Oh yes, I forget about this part, I gonna remove the GPU part.

and Agree with that ksonnet would be much better

**Pull request merged**